server/app/controllers/sendmail_controller.rb

```diff
@@ -0,0 +1,38 @@
+require 'mailgun'
+
+class SendmailController < ApplicationController
+  def index
+
+    mg_client = Mailgun::Client.new ENV["MAILGUN_API_TOKEN"]
+
+    # Define your message parameters
+    message = "Name: " + params[:content][:name] + "<br />" + "Phone: " + params[:content][:phone] + "<br />" + "Message: <br />" + params[:content][:message]
```

Comment:
use HEREDOC string here http://makandracards.com/makandra/1675-using-heredoc-for-prettier-ruby-code

```
message = <<-MSG
  Name: #{params[:content][:name]}<br />
  Phone: #{params[:content][:phone]}<br />
  Message: <br /> #{params[:content][:message]}
MSG
```